src/bosh_azure_cpi/Gemfile

```diff
@@ -1,14 +1,16 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 
+gem 'azure-storage', '0.12.3.preview'
 gem 'bosh_common',   '1.3262.24.0'
 gem 'bosh_cpi',      '2.4.1'
-gem 'azure-storage', '0.12.3.preview'
-gem 'vhd',           '0.0.4'
 gem 'jwt',           '1.5.6'
+gem 'vhd',           '0.0.4'
 
 group :development, :test do
```

Comment:
You may need to `rubocop` in the group `:development, :test`.
But the package `rubocop` should not be vendored into `vendor/package/`.
Please see the script `vendor_gems`

Reply:
cool.

Reply:
done.